account_financial_report/report/general_ledger.py

```diff
@@ -99,12 +103,12 @@ def _get_initial_balances_pl_ml_domain(
     def _get_accounts_initial_balance(self, initial_domain_bs, initial_domain_pl):
         gl_initial_acc_bs = self.env["account.move.line"].read_group(
             domain=initial_domain_bs,
-            fields=["account_id", "debit", "credit", "balance", "amount_currency"],
+            fields=["account_id", "debit", "credit", "balance", "amount_currency:sum"],
```

Comment:
Why adding `:sum`? Isn't the default operation?

Reply:
In v16 in the `amount_currency` field they have added the parameter `group_operator=None`, I don't know if that's why, but if I don't add the `:sum` it gives an error.